server/src/main/java/io/crate/metadata/IndexReference.java

```diff
@@ -210,6 +239,10 @@ public Map<String, Object> toMapping() {
             mapping.put("analyzer", analyzer);
             mapping.put("type", "text");
         }
+        if (columnNames.isEmpty() == false) {
+            // Relevant only for CREATE TABLE case
+            mapping.put("copy_to", columnNames);
```

Comment:
As mentioned in chat.  This would be the wrong way around,  `columns` are pointing to the indexReference